pspipe_utils/misc.py

```diff
@@ -2,11 +2,59 @@
 Some general utility functions that do not belong to other files.
 """
 
+from pspy import pspy_utils
+from pixell import curvedsky
 
 def str_replace(my_str, old, new):
+    """
+    just like replace but check that the replacement actually happened
+
+    Parameters
+    __________
+    my_str: string
+        the string in which the replacment will happen
+    old: string
+        old part of the string to be replaced
+    new: string
+        what will replace old
+    """
+
     my_new_str = my_str
```

Comment:
I'm pretty sure this line is pointless. Writing `my_new_str = my_str` shares the same reference in memory `id(my_new_str) = id(my_str)`. When using the `replace` method then a new reference/object is created so `my_new_str` is then different. So only kept the next line will be enough.